docs/metrics.md

```diff
@@ -10,6 +10,13 @@ metrics:
   port: 9002
 ```
 
+An example dashboard that can be used with [Grafana](https://grafana.com) can be found at [/contrib/hookshot-dashboard.json](https://github.com/matrix-org/matrix-hookshot/blob/main/contrib/hookshot-dashboard.json).
```

Comment:
Sorry, this is non-obvious: You need to update https://github.com/matrix-org/matrix-hookshot/blob/main/scripts/build-metrics-docs.ts#L26, and then run `build:docs`.

Reply:
Could you clarify the required steps in https://github.com/matrix-org/matrix-hookshot/blob/main/CONTRIBUTING.md ?
I can't simply `yarn build:docs` as I need to install requirements first etc and may not be familiar with that.

Reply:
Should be just `yarn` as mentioned in the setup instructions. The mdbook steps are hidden in the readme, but installing it is just `cargo install mdbook`.

We could probably have a page for developer setup instructions too.